pontoon/base/models.py

```diff
@@ -373,7 +375,8 @@ def locale_checkout_path(self, locale):
         if not self.multi_locale:
             raise ValueError('Cannot get locale_checkout_path for non-multi-locale repos.')
 
-        return os.path.join(self.checkout_path, locale.code)
+        path_components = [self.project.checkout_path] + urlparse(self.url).path.split('/')
+        return os.path.join(*path_components).format(locale_code=locale.code)
```

Comment:
Why this change? `checkout_path` already includes path components from the URL and we append the URL to the end of the path instead of formatting intentionally (so that the locale code directory is at the top level of the checkout).